packages/insomnia/src/routes/organization.$organizationId.project.$projectId.workspace.$workspaceId.debug.request.new.tsx

```diff
@@ -123,7 +123,33 @@ export async function clientAction({ params, request }: Route.ClientActionArgs)
   }
   invariant(typeof activeRequestId === 'string', 'Request ID is required');
   models.stats.incrementCreatedRequests();
-  window.main.trackSegmentEvent({ event: SegmentEvent.requestCreate, properties: { requestType } });
+
+  const certificates = await models.clientCertificate.findByParentId(workspaceId);
+
+  window.main.trackSegmentEvent({
+    event: SegmentEvent.requestCreated,
+    properties: {
+      requestType,
+      protocol: requestType,
+      has_prescript: !!req?.preRequestScript,
+      has_postscript: !!req?.afterResponseScript,
+      request_header_names: req?.headers?.map(h => h.name) || [],
+      count_cookies: req?.headers.find(h => h.name.toLowerCase() === 'cookie')
+        ? req.headers.find(h => h.name.toLowerCase() === 'cookie')!.value.split(';').length
```

Comment:
```suggestion
        ? req.headers.find(h => h.name.toLowerCase() === 'cookie')?.value.split(';').length
```

Is there a case where this will not be available? If so it might throw an error